examples/brightness/main.py

```diff
@@ -0,0 +1,41 @@
+from kivy.app import App
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.slider import Slider
+from kivy.lang import Builder
+from plyer import brightness
+from kivy.uix.button import Button
+from kivy.properties import NumericProperty
+
+Builder.load_string('''
+<BrightnessInterface>:
+    orientation: 'vertical'
+    Label:
+        text: 'Adjust the slider to increase \\n or decrease the brightness'
+    Slider:
+        id: slider
+        min: 0
+        max: 100
+        value: root.get_current_brightness()
+        on_touch_up: root.set_brightness(slider.value)
```

Comment:
you might want to try using on_value to get brightness to update while the slider is moving.